### PR TITLE
Ajpal/track sort

### PR DESCRIPTION
**Pull request by ajpal**
@dsjen 
Add tracking for sort.

### CODE REVIEW

File: analytics_dashboard/static/apps/learners/app/app.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ define(function (require) {
                     rootView: rootView,
                     learnerEngagementTimelineUrl: this.options.learnerEngagementTimelineUrl,
                     learnerListUrl: this.options.learnerListUrl,
-                    trackingModel: initModels.trackingModel
+                    trackingModel: initModels.models.trackingModel
```
Comment (dsjen) [inline analytics_dashboard/static/apps/learners/app/app.js:81]: YES!

File: analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js [hunk 2]
```diff
@@ -384,6 +384,27 @@ define(function (require) {
                 getLastRequest().respond(200, {}, JSON.stringify(getResponseBody(2, 1)));
                 expect(this.rosterView.$('a[title="Page 1"]').parent('li')).toHaveClass('active');
             });
+
+            it ('triggers a tracking event', function() {
+                var triggerSpy = spyOn(this.rosterView.options.trackingModel, 'trigger'),
+                headerClasses = [
```
Comment (dsjen) [inline analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js]: Please indent the this and the following lines (our JS convention).

**Comment by dsjen** [comment]
Great test!  Just one indentation request.  👍

**Comment by dsjen** [comment]
FYI, @thallada

File: analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js (hunk 2 above)
Comment (ajpal) [inline analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js]: 👍

**Pull request merged**